src/main/src/main/java/org/geoserver/catalog/ResourcePool.java

```diff
@@ -582,6 +588,12 @@ public DataAccessFactory getDataStoreFactory(DataStoreInfo info) throws IOExcept
         }
     }
 
+    private void disableStoreInfo(WorkspaceInfo ws, String name) {
```

Comment:
The documentation, GUI and REST configuration work covers all types of store. And yet, the ResoucePool/ResourcePoolTest changes seem to only cover DataStore implementations, I don't see the equivalent for the other types of store.